### PR TITLE
Update to Outdated files in the dev-1.18-ko.4 branch.

### LINKED ISSUE #20987: Outdated files in the dev-1.18-ko.4 branch
# This is a Bug Report
## Problem
Outdated files in the dev-1.18-ko.4 branch.

### 54 files to be modified 
1. [ ] content/en/_index.html 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/architecture/nodes.md 1 file changed, 172 insertions(+), 147 deletions(-)
1. [ ] content/en/docs/concepts/cluster-administration/addons.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/concepts/cluster-administration/cloud-providers.md 1 file changed, 22 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/cluster-administration/networking.md 1 file changed, 4 insertions(+)
1. [ ] content/en/docs/concepts/configuration/configmap.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/configuration/manage-resources-containers.md 1 file changed, 3 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/concepts/configuration/pod-priority-preemption.md 1 file changed, 1 insertion(+), 8 deletions(-)
1. [ ] content/en/docs/concepts/containers/overview.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/containers/runtime-class.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/extend-kubernetes/operator.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/overview/components.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/overview/working-with-objects/labels.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/policy/resource-quotas.md 1 file changed, 2 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/scheduling-eviction/kube-scheduler.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/services-networking/dns-pod-service.md 1 file changed, 3 insertions(+), 5 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/dual-stack.md 1 file changed, 9 insertions(+), 6 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/endpoint-slices.md 1 file changed, 5 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/ingress.md 1 file changed, 12 insertions(+), 13 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/network-policies.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/services-networking/service.md 1 file changed, 12 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/concepts/storage/persistent-volumes.md 1 file changed, 5 insertions(+), 5 deletions(-)
1. [ ] content/en/docs/concepts/storage/storage-classes.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/storage/volumes.md 1 file changed, 7 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/workloads/controllers/daemonset.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/workloads/controllers/deployment.md 1 file changed, 95 insertions(+), 79 deletions(-)
1. [ ] content/en/docs/concepts/workloads/controllers/replicaset.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/pods/disruptions.md 1 file changed, 3 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/pods/init-containers.md 1 file changed, 4 insertions(+), 4 deletions(-)
1. [ ] content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/workloads/pods/podpreset.md 1 file changed, 29 insertions(+), 28 deletions(-)
1. [ ] content/en/docs/contribute/review/for-approvers.md 1 file changed, 1 insertion(+), 2 deletions(-)
1. [ ] content/en/docs/reference/glossary/node.md 1 file changed, 2 insertions(+)
1. [ ] content/en/docs/reference/glossary/taint.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/reference/glossary/toleration.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/setup/learning-environment/minikube.md 1 file changed, 126 insertions(+), 90 deletions(-)
1. [ ] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 109 insertions(+), 34 deletions(-)
1. [ ] content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/access-application-cluster/access-cluster.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md 1 file changed, 68 insertions(+), 68 deletions(-)
1. [ ] content/en/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md 1 file changed, 2 insertions(+), 4 deletions(-)
1. [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/tasks/tools/install-kubectl.md 1 file changed, 39 insertions(+), 20 deletions(-)
1. [ ] content/en/docs/tutorials/clusters/apparmor.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tutorials/hello-minikube.md 1 file changed, 6 insertions(+), 12 deletions(-)
1. [ ] content/en/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md 1 file changed, 13 insertions(+), 9 deletions(-)
1. [ ] content/en/docs/tutorials/stateless-application/guestbook.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/examples/application/hpa/php-apache.yaml 1 file changed, 1 deletion(-)

### 2 files to be renamed  
1. [ ] R096 content/en/docs/concepts/configuration/assign-pod-node.md -> content/en/docs/concepts/scheduling-eviction/assign-pod-node.md
1. [ ] R082 content/en/docs/concepts/configuration/taint-and-toleration.md -> content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

### 42 files to be deleted 
1. [ ] content/en/case-studies/ccp-games/ccp_logo.png
1. [ ] content/en/case-studies/ccp-games/index.html
1. [ ] content/en/case-studies/comcast/comcast_logo.png
1. [ ] content/en/case-studies/comcast/index.html
1. [ ] content/en/case-studies/concur/concur_featured_logo.png
1. [ ] content/en/case-studies/concur/index.html
1. [ ] content/en/case-studies/ebay/ebay_featured.png
1. [ ] content/en/case-studies/ebay/ebay_logo.png
1. [ ] content/en/case-studies/ebay/index.html
1. [ ] content/en/case-studies/goldman-sachs/gs_logo.png
1. [ ] content/en/case-studies/goldman-sachs/index.html
1. [ ] content/en/case-studies/homeoffice/homeoffice_logo.png
1. [ ] content/en/case-studies/homeoffice/index.html
1. [ ] content/en/case-studies/jd/index.html
1. [ ] content/en/case-studies/jd/jd_logo.png
1. [ ] content/en/case-studies/liveperson/index.html
1. [ ] content/en/case-studies/liveperson/liveperson_logo.png
1. [ ] content/en/case-studies/monzo/index.html
1. [ ] content/en/case-studies/monzo/monzo_logo.png
1. [ ] content/en/case-studies/philips/index.html
1. [ ] content/en/case-studies/philips/philips_logo.png
1. [ ] content/en/case-studies/pokemon-go/index.html
1. [ ] content/en/case-studies/pokemon-go/pokemon_go_logo.png
1. [ ] content/en/case-studies/samsung-sds/index.html
1. [ ] content/en/case-studies/samsung-sds/sds_logo.png
1. [ ] content/en/case-studies/sap/index.html
1. [ ] content/en/case-studies/sap/sap_logo.png
1. [ ] content/en/case-studies/sap/sap_small.png
1. [ ] content/en/case-studies/soundcloud/index.html
1. [ ] content/en/case-studies/soundcloud/soundcloud_logo.png
1. [ ] content/en/case-studies/wepay/index.html
1. [ ] content/en/case-studies/wepay/wepay_logo.png
1. [ ] content/en/case-studies/zulily/index.html
1. [ ] content/en/case-studies/zulily/zulily_featured.png
1. [ ] content/en/case-studies/zulily/zulily_logo.png
1. [ ] content/en/docs/templates/feature-state-alpha.txt
1. [ ] content/en/docs/templates/feature-state-beta.txt
1. [ ] content/en/docs/templates/feature-state-deprecated.txt
1. [ ] content/en/docs/templates/feature-state-stable.txt
1. [ ] content/en/docs/templates/index.md
1. [ ] content/en/examples/minikube/Dockerfile
1. [ ] content/en/examples/minikube/server.js

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `dev-1.18-ko.4` branch.

For example:
```
# checkout `dev-1.18-ko.4`
...
# check what is updated in the upstream 
git diff dev-1.18-ko.3 dev-1.18-ko.4 -- content/en/_index.html
# apply changes to content/ko
vi content/ko/_index.html
...
# commit and push
...
# make PR to `dev-1.18-ko.4`
```

### PR DESCRIPTION
**Pull request by ysyukr**
### 46 of 54 files to be modified
- [x] content/ko/_index.html
- [x] content/ko/docs/concepts/architecture/nodes.md
- [x] content/ko/docs/concepts/cluster-administration/addons.md
- [x] content/ko/docs/concepts/cluster-administration/cloud-providers.md
- [x] content/ko/docs/concepts/cluster-administration/networking.md
- [x] content/ko/docs/concepts/configuration/configmap.md
- [x] content/ko/docs/concepts/configuration/manage-resources-containers.md
- [x] content/ko/docs/concepts/configuration/pod-priority-preemption.md
- [ ] content/ko/docs/concepts/containers/overview.md
> 오타수정으로 번역시 영향이 없어 미반영
- [x] content/ko/docs/concepts/containers/runtime-class.md
- [ ] content/ko/docs/concepts/extend-kubernetes/operator.md
> fragment identifier 수정이지만, 해당 부분은 한국어로 반영되어 있어 영향이 없어 미반영
- [ ] content/ko/docs/concepts/overview/components.md
> 소제목의 대소문자 변경으로 번역시 영향이 없어 미반영
- [x] content/ko/docs/concepts/overview/working-with-objects/labels.md
- [x] content/ko/docs/concepts/policy/resource-quotas.md
- [x] content/ko/docs/concepts/scheduling-eviction/kube-scheduler.md
- [x] content/ko/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md
- [x] content/ko/docs/concepts/services-networking/dns-pod-service.md
- [x] content/ko/docs/concepts/services-networking/dual-stack.md
- [x] content/ko/docs/concepts/services-networking/endpoint-slices.md
- [x] content/ko/docs/concepts/services-networking/ingress.md
- [ ] content/ko/docs/concepts/services-networking/network-policies.md
> 변경점 반영시 영향이 없어 미반영
- [x] content/ko/docs/concepts/services-networking/service.md
- [x] content/ko/docs/concepts/storage/persistent-volumes.md
- [x] content/ko/docs/concepts/storage/storage-classes.md
- [x] content/ko/docs/concepts/storage/volumes.md
- [x] content/ko/docs/concepts/workloads/controllers/daemonset.md
- [x] content/ko/docs/concepts/workloads/controllers/deployment.md
- [x] content/ko/docs/concepts/workloads/controllers/replicaset.md
- [x] content/ko/docs/concepts/workloads/pods/disruptions.md
- [x] content/ko/docs/concepts/workloads/pods/init-containers.md
- [x] content/ko/docs/concepts/workloads/pods/pod-topology-spread-constraints.md
- [x] content/ko/docs/concepts/workloads/pods/podpreset.md
- [x] content/ko/docs/contribute/review/for-approvers.md
- [x] content/ko/docs/reference/glossary/node.md
- [x] content/ko/docs/reference/glossary/taint.md
- [x] content/ko/docs/reference/glossary/toleration.md
- [x] content/ko/docs/setup/learning-environment/minikube.md
- [x] content/ko/docs/setup/production-environment/container-runtimes.md
- [x] content/ko/docs/setup/production-environment/windows/user-guide-windows-containers.md
- [x] content/ko/docs/tasks/access-application-cluster/access-cluster.md
- [x] content/ko/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md
- [x] content/ko/docs/tasks/access-application-cluster/web-ui-dashboard.md
- [ ] content/ko/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md
> 레이아웃 문제로 #21076 으로 master 브랜치로 선 반영 조치 진행
- [x] content/ko/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md
- [x] content/ko/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md
- [x] content/ko/docs/tasks/manage-kubernetes-objects/declarative-config.md
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md
- [ ] content/ko/docs/tasks/run-application/horizontal-pod-autoscale.md
> 오타수정으로 번역시 영향이 없어 미반영
- [x] content/ko/docs/tasks/tools/install-kubectl.md
- [x] content/ko/docs/tutorials/clusters/apparmor.md
- [x] content/ko/docs/tutorials/hello-minikube.md
- [x] content/ko/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md
- [ ] content/ko/docs/tutorials/stateless-application/guestbook.md
> 문법 수정으로 번역시 영향이 없어 미반영
- [ ] content/ko/examples/application/hpa/php-apache.yaml
> #20555 에서 반영됨

### 2 of 2 files to be renamed
- [x] R096 content/en/docs/concepts/configuration/assign-pod-node.md -> content/en/docs/concepts/scheduling-eviction/assign-pod-node.md
- [x] R082 content/en/docs/concepts/configuration/taint-and-toleration.md -> content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

### 42 of 42 files to be deleted
- [x] content/en/case-studies/ccp-games/ccp_logo.png
- [x] content/en/case-studies/ccp-games/index.html
- [x] content/en/case-studies/comcast/comcast_logo.png
- [x] content/en/case-studies/comcast/index.html
- [x] content/en/case-studies/concur/concur_featured_logo.png
- [x] content/en/case-studies/concur/index.html
- [x] content/en/case-studies/ebay/ebay_featured.png
- [x] content/en/case-studies/ebay/ebay_logo.png
- [x] content/en/case-studies/ebay/index.html
- [x] content/en/case-studies/goldman-sachs/gs_logo.png
- [x] content/en/case-studies/goldman-sachs/index.html
- [x] content/en/case-studies/homeoffice/homeoffice_logo.png
- [x] content/en/case-studies/homeoffice/index.html
- [x] content/en/case-studies/jd/index.html
- [x] content/en/case-studies/jd/jd_logo.png
- [x] content/en/case-studies/liveperson/index.html
- [x] content/en/case-studies/liveperson/liveperson_logo.png
- [x] content/en/case-studies/monzo/index.html
- [x] content/en/case-studies/monzo/monzo_logo.png
- [x] content/en/case-studies/philips/index.html
- [x] content/en/case-studies/philips/philips_logo.png
- [x] content/en/case-studies/pokemon-go/index.html
- [x] content/en/case-studies/pokemon-go/pokemon_go_logo.png
- [x] content/en/case-studies/samsung-sds/index.html
- [x] content/en/case-studies/samsung-sds/sds_logo.png
- [x] content/en/case-studies/sap/index.html
- [x] content/en/case-studies/sap/sap_logo.png
- [x] content/en/case-studies/sap/sap_small.png
- [x] content/en/case-studies/soundcloud/index.html
- [x] content/en/case-studies/soundcloud/soundcloud_logo.png
- [x] content/en/case-studies/wepay/index.html
- [x] content/en/case-studies/wepay/wepay_logo.png
- [x] content/en/case-studies/zulily/index.html
- [x] content/en/case-studies/zulily/zulily_featured.png
- [x] content/en/case-studies/zulily/zulily_logo.png
- [x] content/en/docs/templates/feature-state-alpha.txt
- [x] content/en/docs/templates/feature-state-beta.txt
- [x] content/en/docs/templates/feature-state-deprecated.txt
- [x] content/en/docs/templates/feature-state-stable.txt
- [x] content/en/docs/templates/index.md
- [x] content/en/examples/minikube/Dockerfile
- [x] content/en/examples/minikube/server.js

Fix #20987
/language ko

### CODE REVIEW

**Comment by ysyukr** [comment]
#20554 에서 누락되었던 항목이 발견되어 추가 반영했습니다.
1 files to be deleted
- [x] content/ko/docs/concepts/architecture/master-node-communication.md

File: content/ko/docs/concepts/workloads/controllers/deployment.md [hunk 1]
```diff
@@ -65,84 +66,93 @@ _디플로이먼트_ 는 [파드](/ko/docs/concepts/workloads/pods/pod/)와
...
+
+6. 각 파드에 자동으로 생성된 레이블을 보려면 `kubectl get pods --show-labels` 를 실행한다.
+   다음과 유사하게 출력된다.
+   ```shell
+   NAME                                READY     STATUS    RESTARTS   AGE       LABELS
+   nginx-deployment-75675f5897-7ci7o   1/1       Running   0          18s       app=nginx,pod-template-hash=3123191453
+   nginx-deployment-75675f5897-kzszj   1/1       Running   0          18s       app=nginx,pod-template-hash=3123191453
+   nginx-deployment-75675f5897-qqcnn   1/1       Running   0          18s       app=nginx,pod-template-hash=3123191453
+   ```
+   만들어진 레플리카셋은 3개의 `nginx` 파드가 있는 것을 보장한다.
```
Comment (pjhwa) [inline content/ko/docs/concepts/workloads/controllers/deployment.md:142]:
```suggestion
   만들어진 레플리카셋은 실행 중인 3개의 `nginx` 파드를 보장한다.
```

- 의역을 해보았습니다. 참고하세요.

File: content/ko/docs/concepts/workloads/controllers/deployment.md [hunk 2]
```diff
@@ -1080,6 +1088,15 @@ API 버전 `apps/v1` 에서는 `.spec.selector` 와 `.metadata.labels` 이 설
 
 기존의 모든 파드는 `.spec.strategy.type==Recreate` 이면 새 파드가 생성되기 전에 죽는다.
 
+{{< note >}}
+이렇게 하면 업그레이드를 생성하기 전에 파드 종료를 보장할 수 있다. 디플로이먼트를 업그레이드하면,
+이전 버전의 모든 파드가 즉시 종료된다. 신규 버전의 파드가 생성되기 전에 성공적으로 제거가
+완료되기를 대기한다. 파드를 수동으로 삭제하면 레플리카셋에 의해 라이프사이클로
+제어되며(이전 파드가 여전히 종료 상태에 있는 경우에도) 교체용 파드가 즉시 생성된다. 파드에
```
Comment (pjhwa) [inline content/ko/docs/concepts/workloads/controllers/deployment.md:1095]:
```suggestion
완료되기를 대기한다. 파드를 수동으로 삭제하면, 라이프사이클은 레플리카셋에 의해
제어되며(이전 파드가 여전히 종료 상태에 있는 경우에도) 교체용 파드가 즉시 생성된다. 파드에
```

- 원문의 의미에 가깝게 수정했습니다.

> If you manually delete a Pod, the lifecycle is controlled by the ReplicaSet and the 
> replacement will be created immediately (even if the old Pod is still in a Terminating state).

File: content/ko/docs/concepts/workloads/controllers/deployment.md [hunk 3]
```diff
@@ -1114,7 +1131,7 @@ API 버전 `apps/v1` 에서는 `.spec.selector` 와 `.metadata.labels` 이 설
 `.spec.progressDeadlineSeconds` 는 디플로어먼트가 표면적으로 `Type=Progressing`, `Status=False`의 
 상태 그리고 리소스가 `Reason=ProgressDeadlineExceeded` 상태로 [진행 실패](#디플로이먼트-실패)를 보고하기 전에 
 디플로이먼트가 진행되는 것을 대기시키는 시간(초)를 명시하는 선택적 필드이다.
-디플로이먼트 컨트롤러는 디플로이먼트를 계속 재시도 한다.
+디플로이먼트 컨트롤러는 디플로이먼트를 계속 재시도 한다. 기본값은 600(초) 이다.
```
Comment (pjhwa) [inline content/ko/docs/concepts/workloads/controllers/deployment.md:1134]:
```suggestion
디플로이먼트 컨트롤러는 디플로이먼트를 계속 재시도 한다. 기본값은 600(초)이다.
```

File: content/ko/docs/concepts/workloads/pods/podpreset.md [hunk 4]
```diff
@@ -5,6 +5,8 @@ weight: 50
 ---
 
 {{% capture overview %}}
+{{< feature-state for_k8s_version="v1.6" state="alpha" >}}
+
 이 페이지는 파드 프리셋에 대한 개요를 제공한다. 파드 프리셋은 파드 생성 시간에 파드에
```
Comment (pjhwa) [inline content/ko/docs/concepts/workloads/pods/podpreset.md:10]:
```suggestion
이 페이지는 파드프리셋(PodPreset)에 대한 개요를 제공한다. 파드프리셋은 파드 생성 시간에 파드에
```

- 문서 처음에 API 오브젝트로 표기되어 한영병기가 좋을 것 같습니다.

File: content/ko/docs/concepts/workloads/pods/podpreset.md [hunk 5]
```diff
@@ -14,16 +16,33 @@ weight: 50
 {{% capture body %}}
 ## 파드 프리셋 이해하기
 
-`Pod Preset`은 파드 생성 시간에 파드에 추가적인 런타임 요구사항을
+파드 프리셋은 파드 생성 시간에 파드에 추가적인 런타임 요구사항을
```
Comment (pjhwa) [inline content/ko/docs/concepts/workloads/pods/podpreset.md:19]:
```suggestion
파드프리셋은 파드 생성 시간에 파드에 추가적인 런타임 요구사항을
```

- 원문에 맞춰 붙여썼습니다.

> A PodPreset is an API resource for injecting additional runtime requirements
> into a Pod at creation time.

File: content/ko/docs/concepts/workloads/pods/podpreset.md [hunk 6]
```diff
@@ -14,16 +16,33 @@ weight: 50
...
+주어진 파드프리셋이 적용되도록 파드에 명시하기 위해서는
 [레이블 셀렉터](/ko/docs/concepts/overview/working-with-objects/labels/#레이블-셀렉터)를 사용한다.
 
-파드 프리셋을 사용하는 것은 파드 템플릿 작성자에게 모든 파드를 위한 모든 정보를 명시적으로
+파드프리셋을 사용하는 것은 파드 템플릿 작성자에게 모든 파드를 위한 모든 정보를 명시적으로
 제공하지는 않아도 되도록 한다. 이렇게 하면, 어떤 특정 서비스를 사용할 파드의 파드
 템플릿 작성자는 해당 서비스에 대한 모든 세부 사항을 알 필요가 없다.
 
-그 배경에 대한 자세한 정보를 위해서는, [파드 프리셋을 위한 디자인 제안](https://git.k8s.io/community/contributors/design-proposals/service-catalog/pod-preset.md)을 참고한다.
+## 클러스터에서 파드 프리셋 활성화하기 {#enable-pod-preset}
```
Comment (pjhwa) [inline content/ko/docs/concepts/workloads/pods/podpreset.md:28]:
```suggestion
## 클러스터에서 파드프리셋 활성화하기 {#enable-pod-preset}
```

File: content/ko/docs/concepts/workloads/pods/podpreset.md [hunk 7]
```diff
@@ -60,31 +79,12 @@ weight: 50
...
-    플래그를 추가한다.
-1.  사용할 네임스페이스 안에서 `PodPreset` 오브젝트를 생성하여
-    파드 프리셋을 정의하였다.
-
 {{% /capture %}}
 
 {{% capture whatsnext %}}
 
-* [파드 프리셋을 사용하여 파드에 데이터 주입하기](/docs/tasks/inject-data-application/podpreset/)
+[파드 프리셋을 사용하여 파드에 데이터 주입하기](/docs/tasks/inject-data-application/podpreset/)를 본다.
```
Comment (pjhwa) [inline content/ko/docs/concepts/workloads/pods/podpreset.md:86]:
```suggestion
[파드프리셋을 사용하여 파드에 데이터 주입하기](/docs/tasks/inject-data-application/podpreset/)를 본다.
```

File: content/ko/docs/concepts/workloads/pods/podpreset.md [hunk 8]
```diff
@@ -60,31 +79,12 @@ weight: 50
...
+
+배경에 대한 자세한 정보를 위해서는, [파드 프리셋을 위한 디자인 제안](https://git.k8s.io/community/contributors/design-proposals/service-catalog/pod-preset.md)을 본다.
```
Comment (pjhwa) [inline content/ko/docs/concepts/workloads/pods/podpreset.md:88]:
```suggestion
배경에 대한 자세한 정보를 위해서는, [파드프리셋을 위한 디자인 제안](https://git.k8s.io/community/contributors/design-proposals/service-catalog/pod-preset.md)을 본다.
```

File: content/ko/docs/setup/learning-environment/minikube.md [hunk 9]
```diff
@@ -33,116 +33,151 @@ Minikube는 다음과 같은 쿠버네티스의 기능을 제공한다.
...
+
+   특정 쿠버네티스 버전, VM, 컨테이너 런타임 상에서 클러스터를 시작하기 위한 보다 상세한 정보는 [클러스터 시작하기](#클러스터-시작하기)를 참조한다.
 
 2. 이제, kubectl을 통해서 클러스터와 상호작용할 수 있다. 보다 상세한 정보는 [클러스터와 상호 작용하기](#클러스터와-상호-작용하기)를 참조한다.
 
-    단순한 HTTP 서버인 `echoserver` 이미지를 사용해서 쿠버네티스 디플로이먼트를 만들고 `--port`를 이용해서 8080 포트로 노출해보자.
-    ```shell
-    kubectl create deployment hello-minikube --image=k8s.gcr.io/echoserver:1.10
-    ```
+   단순한 HTTP 서버인 `echoserver` 이미지를 사용해서 쿠버네티스 디플로이먼트를 만들고 `--port`를 이용해서 8080 포트로 노출해보자.
```
Comment (pjhwa) [inline content/ko/docs/setup/learning-environment/minikube.md:54]:
```suggestion
   간단한 HTTP 서버인 `echoserver` 이미지를 사용해서 쿠버네티스 디플로이먼트를 만들고 `--port`를 이용해서 8080 포트로 노출해보자.
```

- 'simple HTTP server'는 '간단한 HTTP 서버'로 번역할 수도 있지 않을까 합니다. 

>    Let’s create a Kubernetes Deployment using an existing image named `echoserver`, which is a simple HTTP server and expose it on port 8080 using `--port`.

File: content/ko/docs/setup/learning-environment/minikube.md [hunk 10]
```diff
@@ -33,116 +33,151 @@ Minikube는 다음과 같은 쿠버네티스의 기능을 제공한다.
...
-	```
-	NAME                              READY     STATUS              RESTARTS   AGE
-	hello-minikube-3383150820-vctvh   0/1       ContainerCreating   0          3s
-	```
-	출력에서 `STATUS`가 `Running`으로 나타나는 경우, 파드는 이제 떠서 기동 중이다.
-	```
-	NAME                              READY     STATUS    RESTARTS   AGE
-	hello-minikube-3383150820-vctvh   1/1       Running   0          13s
-	```
+   파드가 떠서 구동되고 있는지 확인한다.
```
Comment (pjhwa) [inline content/ko/docs/setup/learning-environment/minikube.md:81]:
```suggestion
   파드가 시작되고 실행 중인지 확인한다.
```

- 좀 더 자연스럽게 수정해봤습니다.

File: content/ko/docs/setup/learning-environment/minikube.md [hunk 11]
```diff
@@ -33,116 +33,151 @@ Minikube는 다음과 같은 쿠버네티스의 기능을 제공한다.
...
+   ```
+
+   출력에서 `STATUS`가 `ContainerCreating`으로 나타나는 경우, 파드는 아직 생성 중이다.
+
+   ```
+   NAME                              READY     STATUS              RESTARTS   AGE
+   hello-minikube-3383150820-vctvh   0/1       ContainerCreating   0          3s
+   ```
+
+   출력에서 `STATUS`가 `Running`으로 나타나는 경우, 파드는 이제 떠서 기동 중이다.
```
Comment (pjhwa) [inline content/ko/docs/setup/learning-environment/minikube.md:94]:
```suggestion
   출력에서 `STATUS`가 `Running`으로 나타나는 경우, 파드는 이제 시작돼서 실행 중이다.
```

File: content/ko/docs/setup/learning-environment/minikube.md [hunk 12]
```diff
@@ -33,116 +33,151 @@ Minikube는 다음과 같은 쿠버네티스의 기능을 제공한다.
...
 5. 서비스 상세를 보기 위해서 노출된 서비스의 URL을 얻는다.
-	```shell
-	minikube service hello-minikube --url
-	```
+
+   ```shell
+   minikube service hello-minikube --url
+   ```
+
 6. 로컬 클러스터의 상세를 보기위해서, 출력에서 얻은 URL을 브라우저에 복사해서 붙여 넣는다.
```
Comment (pjhwa) [inline content/ko/docs/setup/learning-environment/minikube.md:107]:
```suggestion
6. 로컬 클러스터의 상세를 보기위해서, 출력에서 얻은 URL을 브라우저에 복사해서 붙여넣는다.
```

- 표준국어대사전[국립국어원]은 '붙여넣다'를 합성어로 보지 않지만, 이제 문서 따위의 일부를 '오려서 붙이다'의 뜻으로 널리 쓰므로 합성어로 봅니다.

File: content/ko/docs/setup/production-environment/container-runtimes.md [hunk 13]
```diff
@@ -68,29 +68,39 @@ kubelet을 재시작 하는 것은 에러를 해결할 수 없을 것이다.
 시스템에 Docker를 설치하기 위해서 아래의 커맨드들을 사용한다.
```
Comment (pjhwa) [inline content/ko/docs/setup/production-environment/container-runtimes.md:68]:
```suggestion
시스템에 도커를 설치하기 위해서 아래의 커맨드들을 사용한다.
```

- 용어집에 따라 한글화했습니다. 문서에서 일괄 적용이 필요할 것 같습니다.

File: content/ko/docs/setup/production-environment/container-runtimes.md [hunk 14]
```diff
@@ -266,51 +320,75 @@ sysctl --system
...
+```shell
+# (containerd 설치)
 ## 리포지터리 설정
 ### 필요한 패키지 설치
 yum install -y yum-utils device-mapper-persistent-data lvm2
+```
 
-### Docker 리포지터리 추가리
+```shell
+## Docker 리포지터리 추가리
```
Comment (pjhwa) [inline content/ko/docs/setup/production-environment/container-runtimes.md:371]:
```suggestion
## 도커 리포지터리 추가
```

- 오타 수정입니다.

File: content/ko/docs/tutorials/hello-minikube.md [hunk 15]
```diff
@@ -165,7 +159,7 @@ Katacode는 무료로 브라우저에서 쿠버네티스 환경을 제공한다.
 
 5. Katacoda 환경에서만: 서비스 출력에서 `8080`의 반대편에 표시되는 5자리 포트 번호를 기록 한다. 이 포트 번호는 무작위로 생성되며, 사용자마다 다를 수 있다. 포트 번호 텍스트 상자에 포트 번호를 입력한 다음, 포트 표시를 클릭한다. 이전 예시를 사용해서 `30369` 를 입력한다.
 
-    이렇게 하면 당신의 앱을 서비스하는 브라우저 윈도우를 띄우고 "Hello World" 메시지를 보여준다.
+    이렇게 하면 당신의 앱을 서비스하는 브라우저 윈도우를 띄우고 애플리케이션의 응답을 볼수 있다.
```
Comment (pjhwa) [inline content/ko/docs/tutorials/hello-minikube.md:162]:
```suggestion
    이렇게 하면 당신의 앱을 서비스하는 브라우저 윈도우를 띄우고 애플리케이션의 응답을 볼 수 있다.
```

- 띄어쓰기가 필요합니다.

File: content/ko/docs/concepts/workloads/controllers/deployment.md [hunk 16]
```diff
@@ -65,84 +66,93 @@ _디플로이먼트_ 는 [파드](/ko/docs/concepts/workloads/pods/pod/)와
...
 
 ### Pod-template-hash 레이블
 
-{{< note >}}
+{{< caution >}}
 이 레이블은 변경하면 안 된다.
-{{< /note >}}
+{{< /caution >}}
 
 `pod-template-hash` 레이블은 디플로이먼트 컨트롤러에 의해서 디플로이먼트가 생성 또는 채택한 모든 레플리케셋에 추가된다.
```
Comment (jmyung) [inline content/ko/docs/concepts/workloads/controllers/deployment.md:157]:
```suggestion
`pod-template-hash` 레이블은 디플로이먼트 컨트롤러에 의해서 디플로이먼트가 생성 또는 채택한 모든 레플리카셋에 추가된다.
```
오타입니다.

**Pull request merged**